### PR TITLE
Location accuracy

### PR DESCRIPTION
**Pull request by rhod8215**

### CODE REVIEW

**Comment by rhod8215** [comment]
https://drive.google.com/open?id=11u4_yQKa0lc7RDTk5MhrHOW4GkMh8XtR

File: src/app/auth/sign-in/sign-in.page.ts [hunk 1]
```diff
@@ -29,8 +33,32 @@ export class SignInPage implements OnInit {
       email: [null, [Validators.required, Validators.email]],
       password: [null, Validators.required],
     });
+    console.log('[Home] OnInit');
```
Comment (jadurani) [inline src/app/auth/sign-in/sign-in.page.ts]: remove

File: src/app/auth/sign-in/sign-in.page.ts [hunk 2]
```diff
@@ -29,8 +33,32 @@ export class SignInPage implements OnInit {
...
+    this.networkSubscriber();
   }
 
+  async networkSubscriber() {
```
Comment (jadurani) [inline src/app/auth/sign-in/sign-in.page.ts]: place network subscriber in app.component.ts

File: src/app/home/request-disposal/request-disposal.page.ts [hunk 3]
```diff
@@ -38,9 +39,11 @@ export class RequestDisposalPage implements OnInit {
     private locationService: LocationService,
     private router: Router,
     private ngZone: NgZone,
+    private locationAccuracy: LocationAccuracy
   ) { }
 
   ngOnInit() {
+    this.enableLocation();
```
Comment (jadurani) [inline src/app/home/request-disposal/request-disposal.page.ts]: add location checker in app.component.ts

**Pull request merged**